### ISSUE TITLE
Czar Hand

### ISSUE THREAD

**Issue by I3uckwheat** [I3uckwheat/cards-of-carousal#81]
Blocked By: #11 

This component will be used by the players, mostly on mobile devices, to actually play the game when they're the czar. This feature will have responsibilities such as: selecting groups white cards, deselecting groups white cards, and hiding white cards until they have been selected at least once. This component should receive a list of cards and have the ability to notify the wrapper of any changes in card selection. This component will also display which cards are selected by using a wrapper. This component is only the functionality of the selecting and deselecting the cards, it does not include the layout and header.

![image](https://user-images.githubusercontent.com/4411701/108608491-d15bc480-738c-11eb-8ea7-2ba2a4c311fa.png)

It will eventually be used similar to the below:
```js
const [selectedCard, setSelectedCard] = useState(undefined)

return (
  <Parent>
    <CzarHand cardGroups={cardGroups} selectedGroup={Number} onSelect={(selected) => setSelectedCard(selected)} />
  </Parent>
)
```

# User Stories
* As a Czar, I would like to be able to see the groups of cards to select a winner, so I can play the game
* As a Czar, I would like to not be able to see groups of cards until I've tapped on them once, so I can experience the game in the same way as the Players
* As a Czar, I want to be able to select a group of cards and see that I've selected that group with the card-wrapper component 

# Completion Criteria
- [x] Have the following API:
   ```js
   <PlayerHand cards={[Cards]} selectedGroup={index} onSelect={Function} />
   ```
- [x] Accepts the following data structure for `cards`
   ```js
   [
     ["Card One", "Card Two", "Card Three"],
     ["Card Four", "Card Five", "Card Six"],
     ["Card Seven", "Card Eight", "Card Nine"],
   ]
   ```
- [x] Accepts a number for `selectedGroup` notating the index of the group selected.
- [x] fires the `onSelect` callback when a card group is touched, with the proper index of the newly selected card
- [x] Displays a star on the selected card group
- [x] Utilizes the card-wrapper component #11 
- [x] Shows the back of cards until they are tapped, then shows the front
- [x] Shows the front of the card when tapped, and it stays facing up until a new card array is passed
- [x] Displays the selected card as the number noted in the `selected` prop
- [x] Tested
- [x] Uses prop-types
